### ISSUE TITLE
Firefox "too much recursion"

### ISSUE THREAD

**Issue by spikyian** [wout/svg.js#516]
Just upgraded to 2.3.2 to fix a problem of path segments on Google Chrome but now on Firefox I get the "too much recursion" bug. Used to work fine on the previous version I was using (I think 1.0.4).

```
SVG.Element<.create()  svg.js:945
SVG.invent/initializer<()  svg.js:95
SVG.adopt()  svg.js:132
assignNewId()  svg.js:5226
SVG.Element<.extend.clone()  svg.js:997
SVG.BBox<.create()  svg.js:2035
SVG.BBox<.construct.bbox()  svg.js:2068
SVG.BBox<.create()  svg.js:2036
SVG.BBox<.construct.bbox()  svg.js:2068
SVG.BBox<.create()  svg.js:2036
SVG.BBox<.construct.bbox()  svg.js:2068
```
Seems to be caused by using size() on a hidden element:
```
var draw
var panel = {}
draw=SVG('panel').size('100%','1000')// the SVG drawing object
console.log('created draw')
panel.rect = draw.rect(1,1)
panel.rect.hide()
console.log('created rect')
panel.rect.size(155, 103)
console.log('done size')
```

Do I need to unhide before modifying or perhaps wrap the functions so they unhide and rehide as necessary?

**Comment by Fuzzyma** [comment]
2.3.2 was released to get rid of that bug but looks like it made it only worse...
For a quickfix try 2.3.1. However: Using hidden elements is never a good idea because the native `getBBox()` needs a visible element to return something useful.
However I would like to have your code t reproduce that issue

**Comment by Fuzzyma** [comment]
The workaround did not make sure, that the element is made visible before getting its bbox.
Fixed in 2.3.3

**Comment by spikyian** [comment]
2.3.3 seems to have fixed it. Thank you for getting it done so quickly.
